### ISSUE TITLE
Bug - macOS Mojave - Cannot select a chat or a contact from the sidebar. Chatarea says "Select a chat to start messaging"

### ISSUE THREAD

**Issue by Joe1am** [overtake/TelegramSwift#667]
Hi!
I'm not able to select any chat from the sidebar, neither a contact when selecting the contact tab at the bottom. But I can use the right mouse button to open the context menu. There all the options(Pin, Archive, Mute etc.) work as expected.
When on settings tab or an the calls tab, everything works as well.

This happened since upgrading to Version 8. Currently on 8.1 (221723). Same Problem
Running macOs Mojave 10.14.6 as its a Mac Pro without support for newer OS Version.

Like this, the app is useless for me.

**Comment by bora89** [comment]
MacOS 11.5.2 (20G95). I have the same issue - not able to select any chat.

**Comment by bora89** [comment]
Seems like for `8.1.1 (221808) Beta` the issue has been fixed

**Comment by 0338755651** [comment]
0338755651

**Comment by 0338755651** [comment]
[]()

**Comment by Joe1am** [comment]
> Seems like for `8.1.1 (221808) Beta` the issue has been fixed

Can confirm. I'm running 8.1.1 (221825) Beta now. The issue seems to be solved.

**Comment by overtake** [comment]
Yes. It’s fixed in 8.1.1. I’m going to release this version today.

**Comment by bora89** [comment]
No way, in version `8.1.2 (221990) Beta` now a whole app is blank after start

**Comment by bora89** [comment]
![BoraShot 2021-09-30 at 19 09 45@2x](https://user-images.githubusercontent.com/1118710/135491893-0a8f8054-111e-45ba-a5ca-927aa5c12474.png)

**Comment by bora89** [comment]
The same for another build

![BoraShot 2021-09-30 at 23 57 45@2x](https://user-images.githubusercontent.com/1118710/135529123-2e48da8c-eee1-4ceb-be79-32c02dfc4141.png)

**Comment by bora89** [comment]
Completely deleted with app `AppCleaner` and it solved the problem
